docs/plio-for-teams/data-bigquery.md

```diff
@@ -0,0 +1,5 @@
+## Data analysis using BigQuery
+
+Since Plio is a generic tool, we will only prioritize adding dashboards that are useful for majority of our users. It is possible that you might not find some of them useful and/or might want dashboards customized to your use case. To enable you to do that, we update all the data captured from the plios created in your organizational workspace to a BigQuery database specific to your organization every hour. You can connect this BigQuery database with your favourite dashboarding tool like Google Data Studio, etc. to make dashboards specific to your use-case. 
```

Comment:
```suggestion
Since Plio is a generic tool, we will only prioritize adding dashboards to our platform, that are useful for the majority of our users. It is possible that you might not find some of them useful and/or might want dashboards customized to your use case. To enable you to do that, we update all the data captured from the plios created in your organizational workspace to a BigQuery database specific to your organization every hour. You can connect this BigQuery database with your favourite dashboarding tool like Google Data Studio, etc. to make dashboards specific to your use case. 
```